docs/configuration.md

```diff
@@ -41,7 +39,17 @@ directory.
 >environment will remove all parameters related to the old storage
 >configuration.
 
+## Overriding the entire configuration file
+
+If the default configuration is not a sound basis for your usage, or if you are having issues overriding keys from the environment, you can specify an alternate YAML configuration file by mounting it as a volume in the container.
+
+Typically, create a new configuration file from scratch, and call it myconfiguration.yaml, then:
```

Comment:
Should we suggest `myconfiguration.yml` to be consistent with the example? Not sure it really matters.